gamification/tests/test_models.py

```diff
@@ -341,3 +349,64 @@ def test_add_badges_method(self):
 
         # After applying add_badge method
         self.assertEquals(test_participant.badges.count(), 0)
+
+    def test_update_participants_data_with_meta_review_complete(self):
+        # Test the participant who has completed the meta-review activity
+        active_newcomers_list = active_newcomers()
+        active_newcomers_list.extend(['test2', 'test3'])
+        # meta-review participant
+        meta_review_participant.objects.create(login='test2')
+        test_meta_review_participant = meta_review_participant.objects.get(
+                                       login='test2')
```

Comment:
Unnecessary. `meta_review_participant.objects.create(login='test2')` will return the instance.